GetRepoJSON/getRepoJSON.py

```diff
@@ -0,0 +1,37 @@
+# get_repo_json.py
+from time import sleep
+def delayserver():
+    sleep(0.01)
+    return
+
+import os
+from gitea import *
+
+try:
+    URL   = os.environ['ALLSPICE_URL']
+except:
+    print("Invalid URL. Run this command:")
+    print("export ALLSPICE_URL=\"https://hub.allspice.io\"")
```

Comment:
I think it's good that you're not printing and then re-raising, but this also continues running.  Is that intentional?  Do you maybe want to exit here?

Reply:
added quit() to both exception blocks.